tests/release_tests/test_launch/README.md

```diff
@@ -0,0 +1,10 @@
+# Launch Release Tests
+
+These tests are intended to be run before any release of Launch, to ensure common use cases haven't regressed. The tests shouldn't mock anything related to the wandb server, and should make actual API calls the same way a user would.
+
+Prerequisites:
+- You must be logged into a Weights & Biases account that has access to the entity `launch-release-testing`
+- You will also need to have AWS credentials - the env vars `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` should be set
```

Comment:
I really think we just want users to set the contents of the ~/.aws/credentials file and mount that in. 

Devs can get the creds from AWS and populate the file. And use this YAML: https://github.com/wandb/wandb/blob/main/wandb/sdk/launch/deploys/kubernetes/launch-agent.yaml#L46

To apply it.

The secret can be created via:
`kubectl create secret generic aws-secret --from-file=/Users/<uname>/.aws -n<namespace>`